state-chain/amm/src/range_orders.rs

```diff
@@ -176,9 +176,9 @@ pub struct PoolState<LiquidityProvider: Ord> {
 	liquidity_map: BTreeMap<Tick, TickDelta>,
 	positions: BTreeMap<(LiquidityProvider, Tick, Tick), Position>,
 	/// Total fees earned over all time
-	total_fees_earned: SideMap<Amount>,
+	pub(super) total_fees_earned: SideMap<Amount>,
```

Comment:
Not sure about the return value here. Maybe something scalar would make more sense?